src/main/java/de/uhd/ifi/se/decision/management/jira/config/PluginInitializer.java

```diff
@@ -167,15 +168,12 @@ public void createDecisionKnowledgeLinkTypes() {
 		IssueLinkTypeManager issueLinkTypeManager = ComponentAccessor.getComponent(IssueLinkTypeManager.class);
 		List<String> existingIssueLinkTypeNames = getNamesOfExistingIssueLinkTypes();
 
-		if (!existingIssueLinkTypeNames.contains("contain")) {
-			issueLinkTypeManager.createIssueLinkType("contain", "contains", "is contained by", "contain_style");
-		}
-		if (!existingIssueLinkTypeNames.contains("attack")) {
-			issueLinkTypeManager.createIssueLinkType("attack", "attacks", "is attacked by", "contain_style");
-		}
-		if (!existingIssueLinkTypeNames.contains("support")) {
-			issueLinkTypeManager.createIssueLinkType("support", "supports", "is supported by", "contain_style");
+		for (LinkType linkType: LinkType.values()) {
```

Comment:
Please make sure that the default "relates to" relationship by Jira is reused in case it already exists.

Reply:
It is, I think.. the name of the default "relates to" relationship is "Relates" and I named mine the same, so it is not going into the if-Statement